source/atlas-cli-automate.txt

```diff
@@ -0,0 +1,102 @@
+.. _atlas-cli-automate:
+
+=========================================
+Automate Processes with the {+atlas-cli+}
+=========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can use the {+atlas-cli+} in scripts to automate processes. Use the
+following resources and best practices as guidance when you automate
+a process using scripts with the {+atlas-cli+}. 
+
+To learn how to connect to the {+atlas-cli+} programmatically, see
+the :guilabel:`Programmatic User` tabs on :ref:`connect-atlas-cli`.
+
+Resources for Automation with the {+atlas-cli+}
+-----------------------------------------------
+
+.. list-table:: 
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Resource
+     - Objective
+
+   * - :ref:`atlas-cli-env-vars`
+     - Set environment variables that you can define once and use
+       across all of your scripts.
+
+   * - :ref:`go-template-output`
+     - Use Go templates or |json| paths to customize the output from
+       the {+atlas-cli+}. You can include the anticipated custom output in your scripts.
+
+Best Practices for Automation with the {+atlas-cli+}
+----------------------------------------------------
+
+Follow these best practices when you automate processes with the
+{+atlas-cli+}:
```

Comment:
Nit: Not sure we need this sentence, but also will def not die on this hill. Up to you. :)